src/main/java/org/akhq/configs/Ldap.java

```diff
@@ -0,0 +1,56 @@
+package org.akhq.configs;
+
+import io.micronaut.context.annotation.ConfigurationProperties;
+import io.micronaut.core.util.StringUtils;
+import lombok.Data;
+import org.akhq.utils.UserGroupUtils;
+
+import javax.annotation.PostConstruct;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@ConfigurationProperties("akhq.security.ldap")
+@Data
+public class Ldap {
+    private String defaultGroup;
+    private Map<String, GroupMapping> groups = new HashMap<>();
+    private Map<String, UserMapping> users = new HashMap<>();
+    /**
+     * @deprecated Naming isn't consistent with the other mappings
+     */
+    @Deprecated(forRemoval = true)
```

Comment:
Go BC for a good time ! remove it ;) as `user`

Reply:
Alright, if we go breaking that's no problem :+1: